### PR TITLE
Add some stuff to pi_sympy.py

### PR DESCRIPTION
**Pull request by asmeurer**
I plan to fill out the rest, as best I can. The commented out ones don't work.

### CODE REVIEW

File: pi_sympy.py [hunk 1]
```diff
@@ -12,15 +12,48 @@ def pi_001():
...
+    return solve(cos(x), x)[0]*2
+
+# def pi_013():
+#     return simplify(-2*I*LambertW(-pi/2))
+
+# def pi_014():
+#     return solve(besselj(S(1)/2, x), x)[0]
+
+# def pi_015():
+#     return simplify(3*sqrt(3)/2/hyper([(-1,3),(1,3)],[1],1))
```
Comment (asmeurer) [inline pi_sympy.py]: What does this syntax mean? I'm not sure if SymPy recognizes it.

**Comment by fredrik-johansson** [comment]
Cool! The (-1,3) means -1/3, if that's what you're wondering.

**Comment by asmeurer** [comment]
OK, done here.

**Comment by asmeurer** [comment]
By the way, there is no 102.

**Comment by asmeurer** [comment]
Looks like 46 of these work for SymPy. We could probably add some more stuff using the geometry module.

**Comment by fredrik-johansson** [comment]
Thanks! Yes, 101... were just added and 102 turned out to be a duplicate, hence the gap (a replacement is needed).

**Pull request merged**